dist/moment-range.js

```diff
@@ -337,6 +330,67 @@ DateRange.prototype.diff = function(unit) {
   return this.end.diff(this.start, unit);
 };
 
+DateRange.prototype.set = function(start, end) {
+  var parts;
+  var s = start;
+  var e = end;
+
+  if (arguments.length === 1 || end === undefined) {
+    if (typeof start === 'object' && start.length === 2) {
+      s = start[0];
+      e = start[1];
+    }
+    else if (typeof start === 'string') {
+      parts = start.split('/');
+      s = parts[0];
+      e = parts[1];
+    }
+  }
+  
+  if(this.options.limit !== false){
+    this.actualStart = (s === null) ? moment(-8640000000000000) : moment(s);
```

Comment:
8640000000000000 is defined way too many times.

Store this as a constant at the top of the file and reference that variable